notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaRecipient.java

```diff
@@ -69,4 +72,10 @@ public String getUsername() {
     public void setUsername(String username) {
         this.username = username;
     }
+
+    @Override
+    public String toString() {
+        return "JpaRecipient [id=" + id + ", username=" + username + "]";
```

Comment:
would [`org.apache.commons.lang3.builder.ToStringBuilder`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html) be a more effective way to generate this string?